target/consensource-ui
# Services update to TS

## Proposed change/fix

- Add Patrick's `sawtooth` types for signing
- Add `consensource` custom type definitions based on protobufs
- `gitignore` the protobufs directory
- Convert `addressing` with string annotations
- Add service conversions

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

`npm run lint` to see all the errors.
`npm run build` to make sure it at least can build.
